.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           java-version: "17"
 
 
-      # Caching is a very useful part of a CI, as a workflow is executed in a clean environement every time,
+      # Caching is a very useful part of a CI, as a workflow is executed in a clean environment every time,
```

Comment:
Great catch to see this typo ! This will definitely improve code readability :+1: